features/admin.password-recovery-flow-builder.v1/providers/password-recovery-flow-builder-provider.tsx

```diff
@@ -105,27 +110,96 @@ const FlowContextWrapper: FC<PasswordRecoveryFlowBuilderProviderProps> = ({
     const [ selectedAttributes, setSelectedAttributes ] = useState<{ [key: string]: Attribute[] }>({});
     const [ isPublishing, setIsPublishing ] = useState<boolean>(false);
 
+    const requestConfig: RequestConfigInterface = useMemo(() => {
```

Comment:
Shall we reuse this hook [1] here instead?

[1] - https://github.com/wso2/identity-apps/blob/master/features/admin.claims.v1/api/use-get-all-local-claims.ts